controllers/neutronapi_controller.go

```diff
@@ -448,22 +448,21 @@ func (r *NeutronAPIReconciler) reconcileInit(
 			},
 		)
 		if err != nil {
+			if k8s_errors.IsNotFound(err) {
+				instance.Status.Conditions.Set(condition.FalseCondition(
+					condition.TLSInputReadyCondition,
+					condition.RequestedReason,
+					condition.SeverityInfo,
+					fmt.Sprintf(condition.TLSInputReadyWaitingMessage, instance.Spec.TLS.CaBundleSecretName)))
+				return ctrl.Result{}, nil
```

Comment:
Oh, actually...shouldn't this have a `ReqeueAfter` attached?

Reply:
we are watching the cacert, should not be required, or why you think we should have it?